### ISSUE TITLE
Price filter is not updated when active filter is removed

### ISSUE THREAD

**Issue by mikesteeghs** [shellpea/advanced-elasticsuite-catalog#13]
**Steps to reproduce:**

1. Open category page with price filter (as as slider) active
2. Adjust the slider, so the price is used as a filter and click on 'OK'
3. Remove the active price filter individually or by the 'Clear all' button

**Expected result**
The price filter isn't active anymore and the slider is reset to the orignal value.

**Actual result**
The price filter isn't active anymore, but the slider still has the value that was selected before.

Magento version: 2.4.4-p2
PHP version: 7.4
Module version: 1.2.2

**Comment by AlexDroomer** [comment]
@mikesteeghs Hi, thanks for letting me know, I'll look into this issue as soon as possible🙏

**Comment by AlexDroomer** [comment]
@mikesteeghs i added a fix, try to install version 1.2.3
```
composer require "shellpea/magento-advanced-elasticsuite-catalog":"^1.2.3"
```
